content/_data/changelogs/lts.yml

```
@@ -3117,6 +3117,23 @@
     message: |-
       Fix a <em>KeyStores with multiple certificates are not supported</em> error from Jetty when passing certain kinds of certificates, such as domain wildcards (regression in 2.204.3).
 
+- version: 2.204.5
+  date: 2020-03-07
+  changes:
+    - type: major bug
+      references:
+        - url: https://github.com/jenkinsci/winstone/blob/master/CHANGELOG.md#53
+          title: full changelog
+        - title: Jetty 9.4.18 changelog
+          url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.18.v20190429
+        - title: Jetty 9.4.17 changelog
+          url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.17.v20190418
+        - title: Jetty 9.4.16 changelog
+          url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.16.v20190411
```

```suggestion
        - issue: 60409
        - issue: 60199
        - url: https://github.com/jenkinsci/winstone/blob/master/CHANGELOG.md
          title: Winstone changelog
```

Fixes issue references and improper changelog links